src/rust/engine/options/src/parse.rs

```diff
@@ -168,7 +168,7 @@ peg::parser! {
             }
 
         rule list_edits<T>(parse_value: rule<T>) -> Vec<ListEdit<T>>
-            = e:list_edit(&parse_value) ** "," ","? { e }
+            = e:list_edit(&parse_value) ++ "," { e }
```

Comment:
Now we require at least one edit , and we don't support a trailing comma (as the Python parser does not). This prevents ambiguity with `empty_string_string_list()` below.